elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/queryengines/sql/SQLEntityHydrator.java

```diff
@@ -9,78 +9,28 @@
 import com.yahoo.elide.datastores.aggregation.QueryEngine;
 import com.yahoo.elide.datastores.aggregation.query.Query;
 import com.yahoo.elide.datastores.aggregation.queryengines.AbstractEntityHydrator;
-import com.yahoo.elide.utils.coerce.CoerceUtil;
-import lombok.AccessLevel;
-import lombok.Getter;
 
-import java.util.AbstractMap;
-import java.util.Collections;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Map;
-import java.util.stream.Collectors;
-
-import javax.persistence.EntityManager;
+import java.sql.ResultSet;
 
 /**
  * {@link SQLEntityHydrator} hydrates the entity loaded by
  * {@link QueryEngine#executeQuery(Query, QueryEngine.Transaction)}.
  */
 public class SQLEntityHydrator extends AbstractEntityHydrator {
 
-    @Getter(AccessLevel.PRIVATE)
-    private final EntityManager entityManager;
-
     /**
      * Constructor.
      *
      * @param results The loaded objects from {@link QueryEngine#executeQuery(Query, QueryEngine.Transaction)}
      * @param query  The query passed to {@link QueryEngine#executeQuery(Query, QueryEngine.Transaction)} to load the
      *               objects
      * @param entityDictionary  An object that sets entity instance values and provides entity metadata info
-     * @param entityManager  An service that issues JPQL queries to load relationship objects
      */
     public SQLEntityHydrator(
-            List<Object> results,
+            ResultSet results,
```

Comment:
Maybe we can merge this with AbstractEntityHyrdator now that this doesn't do anything.

Reply:
We can just call it EntityHydrator.

Reply:
removed SQLEntityHydrator and renamed AbstractEntityHyrdator -> EntityHydrator